receiver/kubeletstatsreceiver/documentation.md

```diff
@@ -394,6 +394,54 @@ The time since the container started
 | ---- | ----------- | ---------- | ----------------------- | --------- |
 | s | Sum | Int | Cumulative | true |
 
+### k8s.node.runtime.imagefs.available
+
+Node runtime imagefs available
+
+| Unit | Metric Type | Value Type |
+| ---- | ----------- | ---------- |
+| By | Gauge | Int |
+
+### k8s.node.runtime.imagefs.capacity
+
+Node runtime imagefs capacity
+
+| Unit | Metric Type | Value Type |
+| ---- | ----------- | ---------- |
+| By | Gauge | Int |
+
+### k8s.node.runtime.imagefs.inodes
+
+Node runtime imagefs inodes
+
+| Unit | Metric Type | Value Type |
+| ---- | ----------- | ---------- |
+| By | Gauge | Int |
+
+### k8s.node.runtime.imagefs.inodes_free
```

Comment:
I think we should use `k8s.node.runtime.imagefs.inodes.free` to keep the naming in-line with [similar existing metrics](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.84.0/receiver/kubeletstatsreceiver/metadata.yaml#L405) from this receiver.

Reply:
👍 good catch

Reply:
> I think we should use `k8s.node.runtime.imagefs.inodes.free` to keep the naming in-line with [similar existing metrics](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.84.0/receiver/kubeletstatsreceiver/metadata.yaml#L405) from this receiver.

I will solve it later.